### PR TITLE
docs: update SyncVars

### PR DESCRIPTION
**Pull request by phodoval**
Updated/extended SyncVar docs. Merged SyncVar hooks under SyncVars and removed the separate hook page.

### CODE REVIEW

File: doc/Articles/Guides/Sync/SyncVars.md [hunk 1]
```diff
@@ -1,69 +1,52 @@
...
-        // Do your own shot validation here because this runs on the server
-        enemy.GetComponent<Enemy>().health -= 5;
+        // This is executed on the server
+        clickCount++;
     }
 }
 ```
 
-In this example, when a Player clicks on an Enemy, the networked enemy game object is assigned to `PlayerController.currentTarget`. When the player presses X, with a correct target selected, that target is passed through a Command, which runs on the server, to decrement the `health` SyncVar. All clients will be updated with that new value. You can then have a UI on the enemy to show the current value.
+In this example, when a Player A clicks the left mosue button, he sends a RPC to the server where the `clickCount` SyncVar is incremented. All other visible players will be informed about Player A's new `clickCount` value.
```
Comment (Lymdun) [inline doc/Articles/Guides/Sync/SyncVars.md:40]: fix typo

File: doc/Articles/Guides/Sync/SyncVars.md [hunk 2]
```diff
@@ -75,4 +58,61 @@ class Cat : Pet
 
 You can attach the Cat component to your cat prefab, and it will synchronize both it's `name` and `color`.
 
->   **Warning** Both `Cat` and `Pet` should be in the same assembly. If they are in separate assemblies, make sure not to change `name` from inside `Cat` directly, add a method to `Pet` instead. 
+> [!WARNING]
+> Both `Cat` and `Pet` should be in the same assembly. If they are in separate assemblies, make sure not to change `name` from inside `Cat` directly, add a method to `Pet` instead.
```
Comment (Lymdun) [inline doc/Articles/Guides/Sync/SyncVars.md:62]: needs to be confirmed, odds are that this limitation got lifted

**Review by Lymdun — CHANGES_REQUESTED** [review]

File: doc/Articles/Guides/Sync/SyncVars.md (hunk 1 above)
Comment (phodoval) [reply doc/Articles/Guides/Sync/SyncVars.md]: Fixed.

File: doc/Articles/Guides/Sync/SyncVars.md (hunk 2 above)
Comment (phodoval) [reply doc/Articles/Guides/Sync/SyncVars.md:62]: Still applies.

File: doc/Articles/Guides/Sync/SyncVars.md [hunk 3]
```diff
@@ -1,69 +1,52 @@
...
-        // Do your own shot validation here because this runs on the server
-        enemy.GetComponent<Enemy>().health -= 5;
+        // This is executed on the server
+        clickCount++;
     }
 }
 ```
 
-In this example, when a Player clicks on an Enemy, the networked enemy game object is assigned to `PlayerController.currentTarget`. When the player presses X, with a correct target selected, that target is passed through a Command, which runs on the server, to decrement the `health` SyncVar. All clients will be updated with that new value. You can then have a UI on the enemy to show the current value.
+In this example, when Player A clicks the left mouse button, he sends a RPC to the server where the `clickCount` SyncVar is incremented. All other visible players will be informed about Player A's new `clickCount` value.
```
Comment (Lymdun) [inline doc/Articles/Guides/Sync/SyncVars.md:40]: please add a link to Server Rpc article on **RPC**

**Review by Lymdun — CHANGES_REQUESTED** [review]

File: doc/Articles/Guides/Sync/SyncVars.md (hunk 3 above)
Comment (phodoval) [reply doc/Articles/Guides/Sync/SyncVars.md]: Done.

**Pull request merged**